tools/underwater-gps.py

```diff
@@ -66,7 +111,20 @@ def processMasterPosition(response, *args, **kwargs):
         0,                     # alt                       : Current altitude (MSL), in meters (float)
         0                      # climb                     : Current climb rate in meters/second (float)
     )
-    
+
+    # new approach: nmea messages to port 14401
+    try:
+        result = response.json()
+        gga = format(gpgga, datetime.now(), result['lat'], result['lon'])
+        rmc = format(gprmc, datetime.now(), result['lat'], result['lon'], orientation=result['orientation'])
+        vtg = format(gpvtg, orientation=result['orientation'])
+        qgcNmeaSocket.sendto(gga, ('192.168.2.1', 14401))
```

Comment:
can we send `[gga rmc vtg]` and not 3 sendTo calls ?

Reply:
done